css/main.css

```diff
@@ -1,3 +1,222 @@
-/* Add your CSS here */
+/* 
+Fonts
+Headings: Poppins - Semi-bold 600
+https://fonts.google.com/specimen/Poppins
+Body: Quicksand - Light 300
+https://fonts.google.com/specimen/Quicksand
+
+Colors
+Dark beige: #E6DACD
+Light beige: #F4ECE6
+Royal blue: #0150FD
+Black: #000
+White: #FFF */
 
+/* Add your CSS here */
 /* Dont' forget to link this file to your HTML in the <head> */
+:root {
+  --dark-beige: #e6dacd;
+  --light-beige: #f4ece6;
+  --royal-blue: #0150fd;
+  --black: #000;
+  --white: #fff;
+}
+
+body {
+  width: 320px;
+  margin: 0;
+  padding: 0;
+  font-family: "Quicksand", sans-serif;
+}
+/*Headings: font-family: "Poppins", sans-serif; */
+.content {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: auto;
+  grid-template-areas:
+    "header"
+    "intro"
+    "main"
+    "footer";
+}
+
+/* header style */
+header {
+  padding-left: 6%;
+  padding-top: 4%;
+  margin-bottom: 36%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-rows: auto;
+  grid-template-areas:
+    "par par par ham"
+    ". . .  ham";
+}
+.hamburger-menu {
+  grid-area: ham;
+  padding-top: 18%;
+  height: 20px;
+  padding-left: 20px;
+}
+
+main {
+  position: relative;
+  padding-top: 10px;
+}
+.profile-img {
+  display: block;
+  border-radius: 50%;
+  width: 44%;
+  margin: 15px auto 18px auto;
+}
+
+header > p:nth-child(1) {
+  margin: 0;
+  display: grid;
+  grid-area: par;
+  font-family: "Poppins", sans-serif;
+  line-height: 1em;
+  font-size: 20px;
+}
+header > p:nth-child(2) {
+  margin: 0;
+  display: grid;
+  grid-area: par;
+  padding-top: 10%;
+  padding-bottom: 10px;
+  line-height: 2em;
+  letter-spacing: 0.02em;
+  font-size: 14px;
+}
+h1 {
+  font-family: "Poppins", sans-serif;
+}
+.hero {
+  position: relative;
+  background-color: rgb(248, 234, 234);
+  margin-left: 6%;
+  margin-right: 2%;
```

Comment:
How could we centre the hero without using margin left and right? By setting a set width of the container and centring it it would be more reactive? This can be a challenge to do before next week